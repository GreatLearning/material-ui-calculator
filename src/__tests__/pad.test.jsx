import renderer from 'react-test-renderer';
import React from 'react';
import Pad from '../pad';


it('renders correctly', () => {
  const tree = renderer
    .create(<Pad>1</Pad>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
