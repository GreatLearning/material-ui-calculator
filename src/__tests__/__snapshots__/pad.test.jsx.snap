// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Pad-pad-1"
  style={undefined}
>
  <button
    className="MuiButtonBase-root-9 MuiIconButton-root-3 Pad-button-2"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex="-1"
    type="button"
  >
    <span
      className="MuiIconButton-label-8"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "1",
          }
        }
      />
    </span>
    <span
      className="MuiTouchRipple-root-12"
    />
  </button>
</div>
`;
